### PR TITLE
DOCSP-33345: Java comments pt. 8

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-33345
Staging:
Codecs - https://preview-mongodbnorareidy.gatsbyjs.io/java/DOCSP-33345-java-comments-8/fundamentals/data-formats/codecs/
Queries - https://preview-mongodbnorareidy.gatsbyjs.io/java/DOCSP-33345-java-comments-8/fundamentals/crud/query-document/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/includes/fundamentals/code-snippets/Query.java [hunk 1]
```diff
@@ -50,36 +50,51 @@ public static void main(String [] args){
     }
 
     private void comparisonFilter(){
+        // Creates a filter to match documents that have a "qty" value over 7
         // begin comparisonFilter
```
Comment (caitlindavey) [inline source/includes/fundamentals/code-snippets/Query.java:54]:
```suggestion
        // begin comparisonFilter
```

Comment (caitlindavey) [reply source/includes/fundamentals/code-snippets/Query.java:54]: Are these for includes reasons? If so, ignore the suggestion. If not, please remove.

**Review by caitlindavey — APPROVED** [review]
LGTM

File: source/includes/fundamentals/code-snippets/Query.java (hunk 1 above)
Comment (norareidy) [reply source/includes/fundamentals/code-snippets/Query.java:54]: Yes, these are to mark the code to include

**Pull request merged**